lib/dynflow/dispatcher/client_dispatcher.rb

```diff
@@ -80,14 +159,27 @@ def dispatch_response(envelope)
               (on ~Failed do |msg|
                  resolve_tracked_request(envelope.request_id, Dynflow::Error.new(msg.error))
                end),
-              (on Done | Pong do
+              (on Done do
+                 resolve_tracked_request(envelope.request_id)
+               end),
+              (on Pong do
+                 add_ping_cache_record(envelope.sender_id)
```

Comment:
If I read the patch correctly, we're not currently using the client pongs in cache at all. Is that intentional?